data/update.py

```diff
@@ -185,69 +160,66 @@ def scan(options):
     full_command += ["--serial"]
 
   # In real mode, ignore cached data, and parallelize.
-  else:
+  elif not options.get("cache"):
     full_command += ["--force"]
 
   shell_out(full_command)
 
-# Use domain-scan to gather .gov hostnames from public sources.
-# Then run pshtt on each gathered hostname.
-def subdomains(options):
+# Use domain-scan to gather .gov domains from public sources.
+def gather_subdomains(options):
+  print("[gather] Gathering subdomains.")
 
-  # Use domain-scan to gather .gov domains from public sources.
-  def gather_subdomains(gatherer, command):
-    print("[%s][gather] Gathering subdomains." % gatherer)
+  full_command = [GATHER_COMMAND]
 
-    gatherer_output = os.path.join(GATHER_TARGET, gatherer)
+  full_command += [",".join(GATHERER_NAMES)]
+  full_command += GATHERER_OPTIONS
 
-    full_command = [GATHER_COMMAND]
-    full_command += command
+  # Common to all gatherers.
+  # --parents gets auto-included as its own gatherer source.
+  full_command += [
+    "--output=%s" % SUBDOMAIN_DATA_GATHERED,
+    "--suffix=%s" % GATHER_SUFFIX,
+    "--ignore-www",
```

Comment:
`--ignore-www` is a new flag in domain-scan which causes the gatherer to collapse `www.history.example.gov` and `history.example.gov` into the same entry.